sympy/physics/quantum/qubit.py

```diff
@@ -372,6 +396,21 @@ class IntQubit(IntQubitState, Qubit):
 
         >>> Qubit(q)
         |101>
+
+    Please note that ``IntQubit`` also accept ``Qubit``-style list of bits.
```

Comment:
Perhaps "... also accepts a ``Qubit``-style...".

Reply:
thanks. I will fix it.